packages/frontend/src/components/SchedulersView/LogsView.tsx

```diff
@@ -0,0 +1,339 @@
+                                    All jobs
+                                </Text>
+                                <ActionIcon onClick={toggle} size="sm">
+                                    <MantineIcon
+                                        icon={IconChevronDown}
+                                        color="black"
+                                        size={13}
+                                    />
+                                </ActionIcon>
+                            </Group>
+                            <Collapse in={opened}>
+                                <Text fz={12} fw={500} pt="md" color="gray.6">
+                                    {handleScheduledDeliveryLogs(item, logs)[0]
+                                        .task.replace(/([A-Z])/g, ' $1')
+                                        .toLowerCase()}
+                                </Text>
+                                <Text fz={12} fw={500} pt="md" color="gray.6">
+                                    {sendNotificationLogs(item, logs)[0]
+                                        .task.replace(/([A-Z])/g, ' $1')
+                                        .toLowerCase()}
+                                </Text>
+                            </Collapse>
+                        </Box>
+                    ) : (
+                        <Text fz={12} fw={500}>
+                            No jobs yet
+                        </Text>
+                    );
+                },
+            },
+            {
+                id: 'deliveryScheduled',
+                label: 'Delivery scheduled',
+                cell: (item) => {
+                    return currentLogs(item, logs).length > 0 ? (
+                        <Box>
+                            <Group spacing="xxs">
+                                <Text fz={12} fw={500} color="gray.6">
+                                    {formatTime(
+                                        currentLogs(item, logs)[0]
+                                            .scheduledTime,
+                                    )}
+                                </Text>
+                            </Group>
+                            <Collapse in={opened}>
+                                <Text fz={12} fw={500} pt="md" color="gray.6">
+                                    {formatTime(
+                                        handleScheduledDeliveryLogs(
+                                            item,
+                                            logs,
+                                        )[0].scheduledTime,
+                                    )}
+                                </Text>
+                                <Text fz={12} fw={500} pt="md" color="gray.6">
+                                    {formatTime(
+                                        sendNotificationLogs(item, logs)[0]
+                                            .scheduledTime,
+                                    )}
+                                </Text>
```

Comment:
if both of the elements have PT I believe the wrapping Box can become a Stack with sizing